docs/README.md

```diff
@@ -0,0 +1,116 @@
+  This repository holds packages that provide integration between ROS and Ignition.
+  Mainly `ros_ign_package` is being used to launch Gazebo with ROS 2 integration 
+  and `ros_ign_bridge` to convert Gz messages to ROS messages.
+
+* [Igition Gazebo ROS2 Control](https://github.com/ros-controls/gazebo_ros2_control/tree/galactic)
+  This is a ROS 2 package for integrating the ros2_control controller architecture
+  with the Gazebo simulator.
+
+### How to prepare a Robot URDF for using Gazebo with ros2_control
+
+Two main actions needs to be done to prepare the URDF model to use Gazebo and
+ros2_control, configure ros2_control setting using the `ros2_control` URDF
+macro and add a plugin to Gazebo to parses the `ros2_control` tags and 
+loads the appropriate hardware interfaces and controller manager
+
+#### 1. URDF declaration for the [`ros2_control` URDF tag](https://control.ros.org/master/doc/getting_started/getting_started.html#hardware-description-in-urdf)
+
+This is done by adding the xacro macro `ros2_control_panda_arm` (which is 
+integrated from main Panda arm URDF file) and uses the
+[`ign_ros2_control/IgnitionSystem`](https://github.com/ros-controls/gz_ros2_control/blob/master/README.md?plain=1#L93-L118):
+
+* https://github.com/AndrejOrsula/panda_ign_moveit2/blob/master/panda_description/urdf/panda.ros2_control#L28-L30
+
+#### 2. URDF declaration for the Gazebo plugin [IgnitionROS2ControlPlugin](https://github.com/ros-controls/gz_ros2_control/blob/master/README.md?plain=1#L153-L169)
+
+This is done by adding the xacro macro `ign_ros2_control` (which is integrated 
+from main Panda URDF file) and add the Gazebo plugin `IgnitionROS2ControlPlugin`
+that parses the `ros2_control` tags and loads the appropriate hardware interfaces and controller manager
+
+* https://github.com:drejOrsula/panda_ign_moveit2/blob/master/panda_description/urdf/panda.gazebo#L8-L16
+
+### How to connect MoveIt 2 with the Gazebo ros2_control plugins
+
+The example launches MoveIt using igniton_ros2_control:
+
+* [default.launch.py](../launch/default.launch.py#L81-L94)
+
+This snippet is calling the main [Panda MoveIt configuration launch file](https://github.com/AndrejOrsula/panda_ign_moveit2/blob/master/panda_moveit_config/launch/move_group.launch.py):
+
+* Panda main MoveIt 2 move_group invocation:
+  * https://github.com/AndrejOrsula/panda_ign_moveit2/blob/5679c0a97d5e34199271ed7abf02cf303d0e8fb9/panda_moveit_config/launch/move_group.launch.py#L248-L264
+
+* Controller manager
+  * https://github.com/AndrejOrsula/panda_ign_moveit2/blob/5679c0a97d5e34199271ed7abf02cf303d0e8fb9/panda_moveit_config/launch/move_group.launch.py#L309-L324
+
+### How this example declares the target pose for the manipulation
+
+This repository uses a `Gazebo PosePublisher` plugin to indicate the target pose for the manipulation:
+* [follow_target.sdf](../ign_moveit2_examples/worlds/follow_target.sdf#L99-L104)
+
+Translate the pose from Gazebo to ROS using the `ros_ign_bridge`:
```

Comment:
```suggestion
Translate the pose from Gazebo to ROS using the `ros_gz_bridge`:
```

I don't know. I see that the code says ros_ign_bridge, but I feel like inconsistency will confuse people. Maybe the code should be changed too. If we decide to maintain this as an official example, we should eventually update all the names in code.